### PR TITLE
dolt-merge: add page

### PR DESCRIPTION
**Pull request by Torfab**
- [x] The page(s) are in the correct platform directories: `common`, `linux`, `osx`, `windows`, `sunos`, `android`, etc.
- [x] The page(s) have at most 8 examples.
- [x] The page description(s) have links to documentation or a homepage.
- [x] The page(s) follow the [content guidelines](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#guidelines).
- [x] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#commit-message).
- **Version of the command being documented (if known):**

### CODE REVIEW

**Comment by Torfab** [comment]
Pr for #5471 issue

File: pages/common/dolt-merge.md [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+
+`dolt merge --squash {{branch_name}}`
+
+- Create a merge commit even when the merge resolves as a fast-forward:
+
+`dolt merge --no-ff {{branch_name}}`
+
+- Use the given {{message}} as the commit message:
+
+`dolt merge --no-ff -m {{message}} {{branch_name}}`
```
Comment (navarroaxel) [inline pages/common/dolt-merge.md:20]:
```suggestion
`dolt merge --no-ff -m "{{message}}" {{branch_name}}`
```

File: pages/common/dolt-merge.md [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+
+- Incorporate changes from the named commits (since the time their histories diverged from the current branch) into the current branch without updating the commit history:
+
+`dolt merge --squash {{branch_name}}`
+
+- Create a merge commit even when the merge resolves as a fast-forward:
+
+`dolt merge --no-ff {{branch_name}}`
+
+- Use the given {{message}} as the commit message:
```
Comment (navarroaxel) [inline pages/common/dolt-merge.md:18]:
```suggestion
- Merge a branch and create a merge commit with a specific commit message:
```

File: pages/common/dolt-merge.md [hunk 3]
```diff
@@ -0,0 +1,24 @@
...
+
+- Incorporate changes from the named commits (since the time their histories diverged from the current branch) into the current branch:
+
+`dolt merge {{branch_name}}`
+
+- Incorporate changes from the named commits (since the time their histories diverged from the current branch) into the current branch without updating the commit history:
+
+`dolt merge --squash {{branch_name}}`
+
+- Create a merge commit even when the merge resolves as a fast-forward:
```
Comment (navarroaxel) [inline pages/common/dolt-merge.md:14]:
```suggestion
- Merge a branch and create a merge commit even when the merge resolves as a fast-forward:
```

File: pages/common/dolt-merge.md [hunk 4]
```diff
@@ -0,0 +1,24 @@
...
+
+`dolt merge --no-ff {{branch_name}}`
+
+- Use the given {{message}} as the commit message:
+
+`dolt merge --no-ff -m {{message}} {{branch_name}}`
+
+- Abort the current conflict resolution process, and try to reconstruct the pre-merge state:
```
Comment (navarroaxel) [inline pages/common/dolt-merge.md:22]: The `try` verb is not correct, the previous state is always "re-construable"
```suggestion
- Abort the current conflict resolution process:
```

File: pages/common/dolt-merge.md (hunk 1 above)
Comment (Torfab) [reply pages/common/dolt-merge.md]: solved in 3ed2e40c70ce3daaf573c04d92a243a4fe566c7a

File: pages/common/dolt-merge.md (hunk 4 above)
Comment (Torfab) [reply pages/common/dolt-merge.md]: Solved in 3ed2e40c70ce3daaf573c04d92a243a4fe566c7a

File: pages/common/dolt-merge.md (hunk 3 above)
Comment (Torfab) [reply pages/common/dolt-merge.md]: Solved in 3ed2e40c70ce3daaf573c04d92a243a4fe566c7a

File: pages/common/dolt-merge.md (hunk 2 above)
Comment (Torfab) [reply pages/common/dolt-merge.md]: Solved in 3ed2e40c70ce3daaf573c04d92a243a4fe566c7a

File: pages/common/dolt-merge.md [hunk 5]
```diff
@@ -0,0 +1,24 @@
+# dolt merge
+
+> Join two or more development histories together.
+> More information: <https://github.com/dolthub/dolt>.
+
+- Incorporate changes from the named commits (since the time their histories diverged from the current branch) into the current branch:
```
Comment (navarroaxel) [inline pages/common/dolt-merge.md:6]: The branch could be up to date and the command doesn't fail. Also I think we should explain a merge in this page
```suggestion
- Incorporate changes from the named commits into the current branch:
```

Comment (Torfab) [reply pages/common/dolt-merge.md]: solved in 55a842f39a01070f08987fc752e5ab667f8e96ef

File: pages/common/dolt-merge.md [hunk 6]
```diff
@@ -0,0 +1,24 @@
+# dolt merge
+
+> Join two or more development histories together.
+> More information: <https://github.com/dolthub/dolt>.
+
+- Incorporate changes from the named commits into the current branch:
+
+`dolt merge {{branch_name}}`
+
+- Incorporate changes from the named commits (since the time their histories diverged from the current branch) into the current branch without updating the commit history:
```
Comment (navarroaxel) [inline pages/common/dolt-merge.md:10]: I think we shouldn't explain a merge in this page
```suggestion
- Incorporate changes from the named commits into the current branch without updating the commit history:
```

Comment (Torfab) [reply pages/common/dolt-merge.md]: I agree, solved in eb5c002ce1ccfd13c575dc75b90876ccb265adb1

**Review by navarroaxel — APPROVED** [review]

**Review by kbdharun — APPROVED** [review]
LGTM, Thanks for your contribution.

**Pull request merged**